src/Tickets/Seating/app/admin/layouts/style.pcss

```diff
@@ -1 +1,61 @@
-@import "./../../postcss/_tabs.pcss";
\ No newline at end of file
+@import "./../../postcss/_tabs.pcss";
+
+.tec-tickets__tab-content__empty {
```

Comment:
Here and elsewhere: if the classes do not come from existing code, try to avoid using BEM with more than one block separator (`__`).

E.g. rename this class to `.tec-tickets-seating__tab--empty`.
Avoid reproducing the `Very__Long__Class__Names__From__Previous__Versions__Of__The__Plugins`.